webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -35,31 +37,53 @@ const TrialChart: React.FC<Props> = ({
   metricNames,
   metrics,
   onMetricChange,
-  workloads,
   trialId,
+  trialTerminated,
 }: Props) => {
   const [ scale, setScale ] = useState<Scale>(Scale.Linear);
+  const [ trialSumm, setTrialSummary ] = useState<MetricContainer[]>([]);
+
+  const fetchTrialSummary = useCallback(async () => {
+    if (trialId) {
+      const summ = await compareTrials({
```

Comment:
question: why this instead of `summarizeTrials`?

I guess maybe `summarizeTrials` is redundant and we could get rid of it?